From 92f00c07a5cf41e4f475b651d18ec47806630b4a Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Thu, 3 May 2018 16:02:43 +0200
Subject: [PATCH] man: docbook doesn't like line breaks within table cells
 (#8885)

It will pass them on as they are to the formatted man pages, which is
pretty uncool. Let's hence avoid line breaks with table cells.

(cherry picked from commit d225fedb662acb42d2733b0a41607bd4c1e402b6)
---
 man/systemd-detect-virt.xml | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/man/systemd-detect-virt.xml b/man/systemd-detect-virt.xml
index aeeb51d09e..b3ec1a4d03 100644
--- a/man/systemd-detect-virt.xml
+++ b/man/systemd-detect-virt.xml
@@ -98,8 +98,7 @@
 
           <row>
             <entry><varname>kvm</varname></entry>
-            <entry>Linux KVM kernel virtual machine, with whatever software, except
-            Oracle Virtualbox</entry>
+            <entry>Linux KVM kernel virtual machine, with whatever software, except Oracle Virtualbox</entry>
           </row>
 
           <row>
@@ -119,8 +118,7 @@
 
           <row>
             <entry><varname>oracle</varname></entry>
-            <entry>Oracle VM VirtualBox (historically marketed by innotek and Sun Microsystems),
-            for legacy and KVM hypervisor</entry>
+            <entry>Oracle VM VirtualBox (historically marketed by innotek and Sun Microsystems), for legacy and KVM hypervisor</entry>
           </row>
 
           <row>
-- 
2.17.0

